src/main/java/com/Duke/Tasks/Deadline.java

```diff
@@ -0,0 +1,55 @@
+package com.Duke.Tasks;
+
+import com.Duke.TaskManager.DukeException;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/*
+ * This Class encapsulates the command Deadline
+ */
+public class Deadline extends Task{
+    private final LocalDate deadline;
```

Comment:
Maybe "deadline" can be replaced with other name like "deadlineDate" to avoid confusion with the class name?